core/src/main/java/com/graphhopper/routing/InstructionsSurroundingEdges.java

```diff
@@ -89,6 +89,15 @@ public int nrOfPossibleTurns() {
         return 1 + reachableEdges.size();
     }
 
+    /**
+     * Calculates the Number of surrounding edges, including the current turn.
+     * This includes oneways in the wrong direction that are no possible turn.
+     */
```

Comment:
For me "turns" is not really defined and if I think about a definition it will be combination of start and destination edge and then possibleTurns is not correct and should be possibleEdges

Reply:
You are right, it makes reading the code of the instruction generation a bit better understandable I think, but it's not really defined. How about outgoingEdges?

Now we need a way to differentiate the two cases like allowed/passable/traversableOutgoingEdges and allOutgoingEdges? I struggle a bit with possibleEdges, I don't think it describes the situation well enough to understand it without reading the documentation (the same was true for surrounding edges ;)).

Reply:
all and allowedOutgoingEdges sounds good. And yes surrounding edges is also unclear. But naming is really hard :)